nnotepad/js/index.js

```diff
@@ -74,26 +90,33 @@ document.addEventListener('DOMContentLoaded', async (e) => {
     resize.setPointerCapture(e.pointerId);
     const listener = (e) => {
       document.documentElement.style.setProperty(
-          '--input-height', `${e.clientY}px`);
+          '--input-height',
+          `${e.clientY}px`,
```

Comment:
The addition of the header at the top of the page makes this resize calculation incorrect - if you move the resize bar it no longer tracks the mouse, but is offset by the height of the header. Can you fix? Might just be `${e.clientY - e.target.getBoundingClientRect().top}` or something.

Reply:
Thanks, fixed this issue by 

```
document.documentElement.style.setProperty(
          '--input-height',
          `${e.clientY - $('#nav').offsetHeight}px`,
      );
```